code/modules/projectiles/ammunition/energy.dm

```diff
@@ -247,9 +247,17 @@
 
 /obj/item/ammo_casing/energy/toxplasma
```

Comment:
And this one. Also best to rename to avoid confusion.

Reply:
Done